<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DD</class>
 <widget class="QMainWindow" name="DD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DD</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>601</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>input</string>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>37</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="set_iso">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>34</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>device</string>
    </property>
    <widget class="QComboBox" name="inputDev">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>36</y>
       <width>331</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>39</y>
       <width>89</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Write</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>89</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Umount</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="log">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>601</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>601</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>output</string>
    </property>
    <widget class="QTextEdit" name="name_out_file">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Name File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>89</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Read</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>60</y>
       <width>89</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>LS</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
